api/V1/Visits.ts

```diff
@@ -26,6 +26,35 @@ interface AnimalMap {
   [key: string]: VisitSummary;
 }
 
+// getRecordingTag from all tracks of a single recording, get the tag with the
+// highest occurence that isnt unidentified
+function getRecordingTag(tracks: any[], userID: number): TrackTag | null {
+  const animalVote = {};
+  let maxVote = null;
+  for (const track of tracks) {
+    const tag = getTrackTag(track.TrackTags, userID);
+    if (tag) {
+      if (tag.what in animalVote) {
+        animalVote[tag.what].count += 1;
+      } else {
+        animalVote[tag.what] = { tag: tag, count: 1 };
+      }
+      if (
```

Comment:
This would probably be cleaner/clearer if you did the count accumulation as one step, and then just did an ascending sort of `Object.values(animalVote)` on the `count` key and popped off the last value as your result.  It's a bit finicky tracking this inside the accumulation loop IMO.